.rhcicd/clowdapp-engine.yaml

```
@@ -267,6 +267,8 @@ objects:
               name: export-service-psk
               key: psk
               optional: true
+        - name: QUARKUS_LOG_CATEGORY_COM_REDHAT_CLOUD_NOTIFICATIONS_EXPORTS_EXPORTEVENTLISTENER_LEVEL
```

`."` and `".` needs to be replaced with `__` (double underscore) in the env var.

```suggestion
        - name: QUARKUS_LOG_CATEGORY__COM_REDHAT_CLOUD_NOTIFICATIONS_EXPORTS_EXPORTEVENTLISTENER__LEVEL
```

---

Thank you!